app/robots/iCubGenova09/openXRHeadsetParameters.ini

```diff
@@ -52,3 +92,19 @@ horizontal_alignement left
 vertical_alignement   center
 automatically_enabled false
 disable_timeout_in_S  5.0
+
+[SLIDE_0]
+width              0.2
+height             0.2
+x                 -0.08
+y                 -0.2
+z                 -0.5
```

Comment:
Probably we need to tune these since right now they overlap with the realsense

Reply:
Done in https://github.com/robotology/walking-teleoperation/pull/108/commits/58fe6b950f0dd2c39bcb87f7b107b1e0dd3b545d